build/rpms/kubeadm.spec

```diff
@@ -22,14 +22,14 @@ install -p -m 755 -t %{buildroot}%{_bindir} {kubeadm}
 install -p -m 644 -t %{buildroot}%{_sysconfdir}/systemd/system/kubelet.service.d/ {10-kubeadm.conf}
 install -p -m 644 -T {kubelet.env} %{buildroot}%{_sysconfdir}/sysconfig/kubelet
 mkdir -p %{buildroot}%{_libexecdir}/modules-load.d
-mkdir -p %{buildroot}%{_sysctldir}
+mkdir -p %{buildroot}/usr/lib/sysctl.d/
```

Comment:
please put the macro back.  It's supported on EL7+ rpmbuilders. 
Also, that's not the default loc for EL7

Reply:
> Can we assume that the thing building the rpms has access to an rpmbuild that groks the macro? 

Yes.

Reply:
We should always steer to canonical .spec file formats.

Reply:
kk updating